ocs_ci/utility/ibmcloud.py

```diff
@@ -357,7 +358,19 @@ def get_node_by_attached_volume(self, volume):
         else:
             worker_id = out["volume_attachments"][0]["instance"]["name"]
             logger.info(f"volume is  attached to node: {worker_id}")
-            return worker_id
+            worker_nodes = get_nodes(node_type="worker")
+            for worker_node in worker_nodes:
+                logger.info(
+                    f"worker node id is:{worker_node.get()['metadata']['labels']['ibm-cloud.kubernetes.io/worker-id']}"
+                )
+                if (
+                    worker_node.get()["metadata"]["labels"][
+                        "ibm-cloud.kubernetes.io/worker-id"
+                    ]
+                    == worker_id
+                ):
+                    logger.info(f"return worker node is:{worker_node}")
```

Comment:
log worker node name instead of instance

Reply:
Done the changes.